Zotero/Scenes/Detail/ItemDetail/Views/ItemDetailCollectionViewHandler.swift

```diff
@@ -387,10 +289,10 @@ final class ItemDetailCollectionViewHandler: NSObject {
     private func sections(for data: ItemDetailState.Data, isEditing: Bool) -> [Section] {
         if isEditing {
             if data.isAttachment {
-                return [.title, .type, .fields, .dates, .tags, .attachments]
+                return [.title, .type, .fields, .dates]
             } else {
                 // Each section is visible during editing, except dates section. Dates are filled automatically and the user can't change them manually.
```

Comment:
`.dates` seem to be passed below, does this comment need to be updated?

Reply:
Updated in 9c12b1a04cf149a362ee3557080404bfd9cffc3d